proposals/NNNN-noncopyable-generics.md

```diff
@@ -199,12 +214,14 @@ enum List<Elm: ~Copyable>: ~Copyable { /* ... */ }
 enum List<Elm>: ~Copyable where Elm: ~Copyable { /* ... */ }
 ```
 
-Once `Element` does not have a `Copyable` constraint, a noncopyable type like `FileDescriptor` can be substituted in `List`, in addition to copyable ones.
+Sice `Elm` is not required to be `Copyable`, a noncopyable type like `FileDescriptor` can be substituted in `List`, in addition to copyable ones.
```

Comment:
```suggestion
Since `Elm` is not required to be `Copyable`, a noncopyable type like `FileDescriptor` can be substituted in `List`, in addition to copyable ones.
```